apprise/plugins/NotifyMacOSX.py

```diff
@@ -124,19 +126,17 @@ def __init__(self, sound=None, include_image=True, **kwargs):
         Initialize MacOSX Object
         """
 
-        super(NotifyMacOSX, self).__init__(**kwargs)
+        super().__init__(**kwargs)
 
-        # Track whether or not we want to send an image with our notification
-        # or not.
+        # Track whether we want to add an image to the notification.
         self.include_image = include_image
 
-        # Acquire the notify path
+        # Acquire the path to the `terminal-notifier` program.
```

Comment:
You've done this in so many places where you corrected all my comments;  I would be spamming you to applaud your amazing work here.  But let me least acknowledge that i noticed this and it's fantastic! Great work. :rocket: .  I"m busted with my copy/pastes and bad updates in so many spots from other plugins.